Paintroid/src/main/java/org/catrobat/paintroid/common/Constants.java

```diff
@@ -42,6 +42,14 @@
 	public static final String SAVE_QUESTION_FRAGMENT_TAG = "savebeforequitfragment";
 	public static final String CHOOSE_IMAGE_FRAGMENT_TAG = "chooseimagefragment";
 
+	public static final String EXTERNAL_STORAGE_PERMISSION_DIALOG = "externalstoragepermissiondialog";
+
+	public static final int PERMISSION_EXTERNAL_STORAGE = 0;
```

Comment:
please move the `PERMISSION_*` ints to `MainActivityConstants` and group them in an `@interface` (Similar to `CreateFileRequestCode`).
You can then annotate the `requestCode` argument in functions like `handlePermissionRequestResults` with `@PermissionRequestCode` (or how ever you name the interface)